### PR TITLE
Added Ankr as Endpoint Provider

### PR DESCRIPTION
**Pull request by kaymomin**
- Added Ankr's RPC in the docs
- Added a compressed image

### CODE REVIEW

**Comment by albertov19** [comment]
Thanks @kaymomin  - Will Ankr support Moonriver and Moonbase Alpha?

**Comment by albertov19** [comment]
@kaymomin It would be preferable to add the public endpoint and endpoint info once you support all networks

**Comment by albertov19** [comment]
@kaymomin we prefer to wait until you have endpoints for all networks. Thanks for understanding

**Comment by kaymomin** [comment]
@albertov19 just following up! There's a possibility that this might be longer as it relates more on those chains collaborating with Ankr directly.

My suggestion is either, we add what's already available or we close this PR, but I think it would be a disservice to developers to not be able to give them options.

Cheers!

**Comment by albertov19** [comment]
@kaymomin is it possible to at least offer an endpoint for Moonriver as well?

**Comment by albertov19** [comment]
@kaymomin could you merge master and also organize Ankr alphabetically in both the list and the actual section. Thanks

**Comment by albertov19** [comment]
@kaymomin you still need to fix conflicts 😉

**Comment by albertov19** [comment]
Also @kaymomin I noticed that you provide a public endpoint. You should add it (in alphabetical order) in: https://github.com/PureStake/moonbeam-docs/blob/master/.snippets/text/endpoints/moonbeam.md

If you also have a WSS endpoint (which would be very useful), please add it there.

File: builders/get-started/endpoints.md [hunk 1]
```diff
@@ -31,11 +31,26 @@ If you are looking for an API provider suitable for production use, you can chec
...
 - [Blast](#blast)
 - [Dwellir](#dwellir)
 - [OnFinality](#onfinality)
 - [Pocket Network](#pokt)
 
+### Ankr {: #ankr}
+
+[Ankr](https://www.ankr.com/){target=_blank} supports free, public RPC endpoints to 15 different blockchain ecosystems, with additional networks constantly being added. The Ankr Public RPC layer provides fast and reliable access via API endpoints for anybody in the world to connect to a growing number of networks including  Moonbeam. 
+
+To get started, head to the [Ankr Protocol](https://www.ankr.com/protocol/) page to make your first call!
```
Comment (eshaben) [inline builders/get-started/endpoints.md:44]:
```suggestion
To get started, head to the [Ankr Protocol](https://www.ankr.com/protocol/){target=_blank} page to make your first call!
```

File: builders/get-started/endpoints.md [hunk 2]
```diff
@@ -31,11 +31,26 @@ If you are looking for an API provider suitable for production use, you can chec
...
+
+1. Click on **Public RPCs**
+2. Select the [Moonbeam Network](https://www.ankr.com/protocol/public/moonbeam/)
```
Comment (eshaben) [inline builders/get-started/endpoints.md:47]:
```suggestion
2. Select the [Moonbeam Network](https://www.ankr.com/protocol/public/moonbeam/){target=_blank}
```

File: builders/get-started/endpoints.md [hunk 3]
```diff
@@ -31,11 +31,26 @@ If you are looking for an API provider suitable for production use, you can chec
...
 
+- [Ankr](#ankr)
 - [Blast](#blast)
 - [Dwellir](#dwellir)
 - [OnFinality](#onfinality)
 - [Pocket Network](#pokt)
 
+### Ankr {: #ankr}
+
+[Ankr](https://www.ankr.com/){target=_blank} supports free, public RPC endpoints to 15 different blockchain ecosystems, with additional networks constantly being added. The Ankr Public RPC layer provides fast and reliable access via API endpoints for anybody in the world to connect to a growing number of networks including  Moonbeam.
```
Comment (eshaben) [inline builders/get-started/endpoints.md:42]:
```suggestion
[Ankr](https://www.ankr.com/){target=_blank} supports free, public RPC endpoints to 15 different blockchain ecosystems, with additional networks constantly being added. The Ankr public RPC layer provides fast and reliable access via API endpoints for anybody in the world to connect to a growing number of networks including Moonbeam. 
```

**Review by eshaben — CHANGES_REQUESTED** [review]
Just a couple minor changes, and then as Alberto had suggested, you can also add the endpoint to this file: https://github.com/PureStake/moonbeam-docs/blob/master/.snippets/text/endpoints/moonbeam.md 🙂

**Comment by albertov19** [comment]
@eshaben FYI - they added the other part in a separate PR -> https://github.com/PureStake/moonbeam-docs/pull/350

**Review by eshaben — APPROVED** [review]

**Pull request merged**